teuthology/suite/__init__.py

```diff
@@ -25,7 +25,6 @@ def override_arg_defaults(name, default, env=os.environ):
         '--suite-repo'        : 'TEUTH_SUITE_REPO',
         '--ceph-branch'       : 'TEUTH_CEPH_BRANCH',
         '--suite-branch'      : 'TEUTH_SUITE_BRANCH',
-        '--teuthology-branch' : 'TEUTH_BRANCH',
```

Comment:
I'm probably missing something obvious, but what is the reason for getting rid of this and doing os.environ.get() later?

(Same question for `default_teuthology_branch` above.)

Reply:
this overall code and `default_teuthology_branch` was introduced to show the default value for `-t` option, but since the algorithm of determination (which is described in the `-t`'s help now)  is changed and in some cases requires to literally checkout suite-repo, there is no sense to do that, because it will confuse the user.